FLEDGE.md

```diff
@@ -803,3 +811,184 @@ These signals were requested in [issue 435](https://github.com/WICG/turtledove/i
+});
+```
+
+#### 6.1. Auction Nonce
+
+To prevent unintended replaying of additional bids, any auction config, whether top-level or component auction config, must include an auction nonce value if it includes additional bids.  The auction nonce is fetched provided like so:
+
+```
+const auctionNonce = navigator.createAuctionNonce();
+navigator.runAdAuction({
+  // ...
+  'auctionNonce':  auctionNonce,
+  'additionalBids': ...,
+});
+```
+
+The same nonce value will need to appear in the auctionNonce field of each additional bid associated with that auction config. Auctions that don't use additional bids don't need to create or provide an auction nonce.
+
+#### 6.2. Negative Targeting
+
+In online ad auctions for ad space, it’s sometimes useful to prevent showing an ad to certain audiences, a concept known as negative targeting. For example, you might not want to show a new customer advertisement to existing customers. New customer acquisition campaigns most often have this as a critical requirement.
+
+To facilitate negative targeting in Protected Audience auctions, each additional bid is allowed to identify one or more negative interest groups. If the user has been joined to any of the identified negative interest groups, the additional bid is dropped; otherwise it participates in the auction, competing alongside bids created by calls to generateBid().
+
+##### 6.2.1 Negative Interest Groups
+
+Negative interest groups are joined using the same API as normal interest groups, though a different set of fields must be provided. Notably, only the owner, name, lifetimeMs, updateURL and additionalBidKey fields are allowed for negative interest groups. Conversely, only negative interest groups are allowed to specify the additionalBidKey field. The 'additionalBidKey' field is described in more detail in section 6.3. Additional Bid Keys.
+
+```
+const myGroup = {
+  'owner': 'https://www.example-dsp.com',
+  'name': 'womens-running-shoes',
+  'lifetimeMs': 30 * kSecsPerDay,
+  'updateURL': 'https://www.example-dsp.com/update?id=12345', //optional
+  'additionalBidKey': 'EA/fR/uU8VNqT3w/2ic4P6Azdaj1J8U35vFwPEf5T4Y='
+};
+navigator.joinAdInterestGroup(myGroup);
+```
+
+##### 6.2.2 How Additional Bids Specify their Negative Interest Groups
+
+Additional bids specify the negative interest groups they're negatively targeting against using one of two fields in their JSON data structure - negativeInterestGroup or negativeInterestGroups.
+
+If an additional bid only needs to specify a single negative interest group, it can do so as follows:
+
+```
+const additionalBid = {
+  // ...
+  "negativeInterestGroup": "example_advertiser_negative_interest_group"
+  // ...
+}
+```
+
+If an additional bid needs to specify two or more negative interest groups, all of those negative interest groups must be joined from the same site, and that site must be identified ahead of time in the additional bid using the joiningOrigin field:
```

Comment:
>that **site** must be identified ahead of time in the additional bid using the **joiningOrigin** field

Both site and origin are used. I presume you mean origin since the field is to ensure only a single origin's info is used.

Reply:
Point well taken regarding the distinction between site and origin. The restriction on multiple negative interest groups is that they must be from the same origin. I've updated the wording here to make it clearer.